<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XML Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icons/Icons/XML.png</normaloff>:/Icons/Icons/XML.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">
    TextEdit {
        id: te
        anchors.left: parent.left
        anchors.right: parent.right
        anchors.top: parent.top
    }</string>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoAll</set>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="overwriteMode">
       <bool>true</bool>
      </property>
      <property name="tabStopWidth">
       <number>25</number>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCheck_file"/>
   <addaction name="actionSolve_erorrs"/>
   <addaction name="separator"/>
   <addaction name="actionFormat_file"/>
   <addaction name="actionErase_Spaces"/>
   <addaction name="separator"/>
   <addaction name="actionMake_JSON_file"/>
   <addaction name="actionBack_to_XML_file"/>
   <addaction name="separator"/>
   <addaction name="actionCompress_file"/>
   <addaction name="actionDecompress_file"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose_file"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionSource_Code"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionContact_us"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCheck_file"/>
    <addaction name="actionSolve_erorrs"/>
    <addaction name="separator"/>
    <addaction name="actionFormat_file"/>
    <addaction name="actionErase_Spaces"/>
    <addaction name="separator"/>
    <addaction name="actionMake_JSON_file"/>
    <addaction name="actionBack_to_XML_file"/>
    <addaction name="separator"/>
    <addaction name="actionCompress_file"/>
    <addaction name="actionDecompress_file"/>
   </widget>
   <widget class="QMenu" name="menuText">
    <property name="title">
     <string>Text</string>
    </property>
    <addaction name="actionFont_2"/>
    <addaction name="separator"/>
    <addaction name="actionForeground_Color"/>
    <addaction name="actionBackground_Color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuText"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionClose_file"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_All"/>
   <addaction name="actionFont_2"/>
   <addaction name="actionForeground_Color"/>
   <addaction name="actionBackground_Color"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/new-file.png</normaloff>:/Icons/Icons/new-file.png</iconset>
   </property>
   <property name="text">
    <string>New file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/open-file.png</normaloff>:/Icons/Icons/open-file.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/save-file.png</normaloff>:/Icons/Icons/save-file.png</iconset>
   </property>
   <property name="text">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/Save-as.png</normaloff>:/Icons/Icons/Save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save file as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/exit.png</normaloff>:/Icons/Icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Q</string>
   </property>
  </action>
  <action name="actionClose_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/close-file.png</normaloff>:/Icons/Icons/close-file.png</iconset>
   </property>
   <property name="text">
    <string>Close tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClose_all_files">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/close-files.png</normaloff>:/Icons/Icons/close-files.png</iconset>
   </property>
   <property name="text">
    <string>Close all tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/undo.png</normaloff>:/Icons/Icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/redo.png</normaloff>:/Icons/Icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/files.png</normaloff>:/Icons/Icons/files.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/cut.png</normaloff>:/Icons/Icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/paste.png</normaloff>:/Icons/Icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/find.png</normaloff>:/Icons/Icons/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/replace.png</normaloff>:/Icons/Icons/replace.png</iconset>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFormat_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/copy.png</normaloff>:/Icons/Icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Format file</string>
   </property>
  </action>
  <action name="actionMake_JSON_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/json.png</normaloff>:/Icons/Icons/json.png</iconset>
   </property>
   <property name="text">
    <string>Convert to JSON file</string>
   </property>
  </action>
  <action name="actionCompress_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/compression.png</normaloff>:/Icons/Icons/compression.png</iconset>
   </property>
   <property name="text">
    <string>Compress file</string>
   </property>
  </action>
  <action name="actionSource_Code">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/source-code.png</normaloff>:/Icons/Icons/source-code.png</iconset>
   </property>
   <property name="text">
    <string>Source Code</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/about.png</normaloff>:/Icons/Icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/about-Qt.png</normaloff>:/Icons/Icons/about-Qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionContact_us">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/tutorial.png</normaloff>:/Icons/Icons/tutorial.png</iconset>
   </property>
   <property name="text">
    <string>Tutorials (Help)</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/zoom-in.png</normaloff>:/Icons/Icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/zoom-out.png</normaloff>:/Icons/Icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/select_all.png</normaloff>:/Icons/Icons/select_all.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/color.png</normaloff>:/Icons/Icons/color.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/font.png</normaloff>:/Icons/Icons/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionBold_3">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/text_bold.png</normaloff>:/Icons/Icons/text_bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/text_italic.png</normaloff>:/Icons/Icons/text_italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/text_underline.png</normaloff>:/Icons/Icons/text_underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionCheck_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/diagnostic.png</normaloff>:/Icons/Icons/diagnostic.png</iconset>
   </property>
   <property name="text">
    <string>Check file consistency</string>
   </property>
  </action>
  <action name="actionSolve_erorrs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/Solve problem.png</normaloff>:/Icons/Icons/Solve problem.png</iconset>
   </property>
   <property name="text">
    <string>Solve errors</string>
   </property>
  </action>
  <action name="actionColor_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/color.png</normaloff>:/Icons/Icons/color.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionFont_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/font.png</normaloff>:/Icons/Icons/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionForeground_Color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/color.png</normaloff>:/Icons/Icons/color.png</iconset>
   </property>
   <property name="text">
    <string>Foreground Color</string>
   </property>
  </action>
  <action name="actionBackground_Color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/bucket.png</normaloff>:/Icons/Icons/bucket.png</iconset>
   </property>
   <property name="text">
    <string>Background Color</string>
   </property>
  </action>
  <action name="actionBack_to_XML_file">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/xml Icon.png</normaloff>:/Icons/Icons/xml Icon.png</iconset>
   </property>
   <property name="text">
    <string>Back to XML file</string>
   </property>
  </action>
  <action name="actionErase_Spaces">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/icons8-space-after-paragraph-50.png</normaloff>:/Icons/Icons/icons8-space-after-paragraph-50.png</iconset>
   </property>
   <property name="text">
    <string>Erase Spaces</string>
   </property>
  </action>
  <action name="actionDecompress_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/Icons/zipped-file-icon-15.jpg</normaloff>:/Icons/Icons/zipped-file-icon-15.jpg</iconset>
   </property>
   <property name="text">
    <string>Decompress file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
